.github/workflows/workflow-run-docker-rust-build.yaml

```diff
@@ -73,12 +73,27 @@ permissions:
 
 jobs:
   rust-all:
-    runs-on: experimental-docker
+    runs-on: runs-on,cpu=64,family=c7,hdd=1024,image=aptos-ubuntu-x64,run-id=${{ github.run_id }} 
     steps:
       - uses: actions/checkout@v3
         with:
           ref: ${{ env.GIT_SHA }}
 
+      - name: Setup Runs On Cache for Docker
+        uses: runs-on/cache@v4
+        with:
+          path: |
+            /home/runner/docker-cache.tzst
+          key: docker-buildkit-cache-${{ env.PROFILE }}-${{ env.FEATURES }}${{ hashFiles('Cargo.lock') }}
+
+      - name: Untar cache if present
+        run: |
+          if [ -f /home/runner/docker-cache.tzst ]; then
+            sudo systemctl stop docker
+            sudo tar --posix -xf /home/runner/docker-cache.tzst -P -C /var/lib/docker --use-compress-program zstdmt .
```

Comment:
is this using zstd as algo?

Reply:
I believe so, used the same setup as cache action.